src/main/proto/com/google/api/codegen/config.proto

```diff
@@ -656,3 +658,28 @@ message LongRunningConfigProto {
   // Total polling timeout.
   uint64 total_poll_timeout_millis = 8;
 }
+
+message OutputSpec {
+  message LoopStmt {
+    string collection = 1;
+    string var = 2;
+    repeated OutputSpec body = 3;
+  }
+  message PrintStmt {
+    string spec = 1;
```

Comment:
What if we made the yaml more compact, like this:
```
- print:
  - "Person %s lives at %s\n"
  - response.name
  - response.address
```
which I guess means that `OutputSpec.print` should then be just a `repeated string`. I guess this would mean then an enum instead of a oneof to distinguish the element types.

(Originally I was going to suggest a `repeated string` inside the `PrintStmt`, but IIUC that would still be more verbose in the YAML than what I suggest above)

Reply:
Done